apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/ManageInstances/ManageInstancesDrawer.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import QueuedDrawer from "LLM/components/QueuedDrawer";
+import { TrackScreen } from "~/analytics";
+
+import GenericErrorView from "~/components/GenericErrorView";
+import { Flex, InfiniteLoader } from "@ledgerhq/native-ui";
+
+import { ListInstances } from "../../components/ManageInstances/ListInstances";
+import { useGetMembers } from "../../hooks/useGetMembers";
+import { memberCredentialsSelector } from "@ledgerhq/trustchain/store";
+import { useSelector } from "react-redux";
+import { DeletionError, ErrorReason } from "../../components/ManageInstances/DeletionError";
+
+type Props = {
+  isOpen: boolean;
+  handleClose: () => void;
+};
+
+export enum Scene {
+  List,
+  Instructions,
+  AutoRemove,
+  Unsecured,
+}
+
+const ManageInstancesDrawer = ({ isOpen, handleClose }: Props) => {
+  const { isError, error, isLoading, data } = useGetMembers();
+  const memberCredentials = useSelector(memberCredentialsSelector);
```

Comment:
We should not rely on redux Store inside the view. Could you move it in the viewModel hook, please?